thoth/prescriptions_refresh/handlers/scorecards.py

```diff
@@ -0,0 +1,826 @@
+    )
+
+
+def _handle_sast(
+    prescriptions: "Prescriptions",
+    project_name: str,
+    scorecards_entry: Dict[str, Any],
+) -> None:
+    """Handle SAST scorecards."""
+    if int(scorecards_entry["Confidence"]) == 0:
+        prescriptions.delete_prescription(
+            project_name,
+            prescription_name="scorecards_sast.yaml",
+            commit_message=f"SAST Security Scorecards for {project_name!r} have low confidence",
+            nonexisting_ok=True,
+        )
+        return
+
+    prescription_name = "SAST"
+    for part in map(str.capitalize, project_name.split("-")):
+        prescription_name += part
+    prescription_name += "ScoreCardsWrap"
+
+    if scorecards_entry["Pass"]:
+        justification_type = "INFO"
+        message = "Project uses static source code analysis based on Security Scorecards"
+    else:
+        justification_type = "WARNING"
+        message = "Project does NOT use static source code analysis based on Security Scorecards"
+
+    prescriptions.create_prescription(
+        project_name,
+        "scorecards_sast.yaml",
+        content=_SCORECARDS_WRAP_GENERIC_UNIT.format(
+            prescription_name=prescription_name,
+            package_name=project_name,
+            type=justification_type,
+            message=message,
+        ),
+        commit_message=f"SAST Security Scorecards update for {project_name!r}",
+    )
+
+
+_SCORECARDS_HANDLERS = {
+    "Code-Review": _handle_code_review,
+    "Active": _handle_active,
+    "Maintained": _handle_active,
+    "Automatic-Dependency-Update": _handle_automatic_dependency_update,
+    "Dependency-Update-Tool": _handle_automatic_dependency_update,
+    "Branch-Protection": _handle_branch_protection,
+    "Token-Permissions": _handle_token_permissions,
+    "Security-Policy": _handle_security_policy,
+    "Signed-Releases": _handle_signed_releases,
+    "Signed-Tags": _handle_signed_tags,
+    "Fuzzing": _handle_fuzzing,
+    "Vulnerabilities": _handle_vulnerabilities,
+    "Packaging": _handle_packaging,
+    "Binary-Artifacts": _handle_binary_artifacts,
+    "CII-Best-Practices": _handle_cii_best_practices,
+    "Pinned-Dependencies": _handle_pinned_dependencies,
+    "Frozen-Dependencies": _handle_pinned_dependencies,
+    "Frozen-Deps": _handle_pinned_dependencies,
+    "Contributors": _handle_contributors,
+    "CI-Tests": _handle_ci_tests,
+    "SAST": _handle_sast,
+}
+
+
+def scorecards(prescriptions: "Prescriptions") -> None:
+    """Check scorecards computed by OSSF and create corresponding prescriptions."""
+    # Parse and prepare scorecards in advance.
+    _LOGGER.info("Querying scorecards available in BigQuery")
+    client = bigquery.Client()
+    query_job = client.query("""SELECT * FROM openssf.scorecardcron.scorecard""")
+
+    scorecards_dict = {}
+    for row in query_job:
+        repo = row["Repo"]
+
+        if repo.startswith("github.com/"):
```

Comment:
Should this be part of the query itself, to limit the results?

`SELECT * FROM openssf.scorecardcron.scorecard WHERE STARTSWITH(Repo, "github.com")`